l5-operator/controllers/bestie_controller.go

```diff
@@ -29,12 +29,25 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	// rbacv1 "k8s.io/api/rbac/v1"
```

Comment:
we don’t want our controller or operator creating rbacs that should be done by olm or who ever installs the operator